app/templates/carpools/show.html

```diff
@@ -2,78 +2,163 @@
-</p>
-
-{% set current_user_ride_request = pool.get_current_user_ride_request() %}
-
-
-{% if current_user_ride_request and current_user_ride_request.status == 'approved'  %}
-<p>
-    <b>The driver approved your request</b>
-</p>
-{% endif  %}
-
-
-<p>
-{%- if pool.seats_available > 1 %}
-    There are {{ pool.seats_available }} seats available in this carpool.
-{%- elif pool.seats_available == 1 %}
-    There is 1 seat available in this carpool.
-{%- else %}
-    There are no seats available in this carpool.
-{%- endif %}
-</p>
-
-{% if current_user_ride_request %}
-
-<a class="btn btn-default" href="{{ url_for('carpool.modify_ride_request', carpool_uuid=pool.uuid, request_uuid=current_user_ride_request.uuid, action='cancel') }}" role="button">Cancel your seat request</a>
-
-{% elif pool.current_user_is_driver %}
-
-<a class="btn btn-default" href="{{ url_for('carpool.cancel', uuid=pool.uuid) }}" role="button">Cancel your carpool</a>
-
-<div>
-<h3>Ride Requests</h3>
-<ul>
-{% for request in pool.get_ride_requests_query() %}
-    <li>{{ request.person.name }}, Gender: {{ request.person.gender }}, Notes: {{ request.notes }}&nbsp;<small>
-        {%- if request.status == 'requested' %}
-        (Waiting for your response:&nbsp;
-        <a href="{{ url_for('carpool.modify_ride_request',
-                            action='approve',
-                            carpool_uuid=pool.uuid,
-                            request_uuid=request.uuid) }}">Approve</a>,&nbsp;
-        <a href="{{ url_for('carpool.modify_ride_request',
-                            action='deny',
-                            carpool_uuid=pool.uuid,
-                            request_uuid=request.uuid) }}">Deny</a>)
-        {%- elif request.status == 'approved' %}
-        (You approved their request:&nbsp;
-        <a href="{{ url_for('carpool.modify_ride_request',
-                            action='deny',
-                            carpool_uuid=pool.uuid,
-                            request_uuid=request.uuid) }}">Deny</a>)
-        {%- elif request.status == 'denied' %}
-        (You denied their request:&nbsp;
-        <a href="{{ url_for('carpool.modify_ride_request',
-                            action='approve',
-                            carpool_uuid=pool.uuid,
-                            request_uuid=request.uuid) }}">Approve</a>)
         {%- endif %}
-        </small>
-    </li>
-{% endfor %}
-</ul>
-</div>
 
-{% else %}
-<a class="btn btn-default" href="{{ url_for('carpool.new_rider', carpool_uuid=pool.uuid) }}" role="button">Request a seat</a>
-{% endif %}
+            <div class="two-col-layout top-border">
+                <h4 class=>Driver's Corner: Passenger Requests</h4>
```

Comment:
Missing class here.

Reply:
whoops! moving too fast without being able to run locally this morning! fixed.